SkietbaanBE/SkietbaanBE/Program.cs

```diff
@@ -14,12 +14,13 @@ public class Program
     {
         public static void Main(string[] args)
         {
-            BuildWebHost(args).Run();
+            Console.Write(Environment.GetEnvironmentVariable("TEMP"));
+           // BuildWebHost(args).Run();
```

Comment:
Remove Ghost code please in all instances. Also the additional lines between code please clean that up as well